src/node/desktop/forge.config.js

```diff
@@ -1,3 +1,12 @@
+const { createFullPackageFileName } = require('./scripts/create-full-package-file-name.js');
+// This function makes sure that the correct filename is created and saved for the final DMG file.
+createFullPackageFileName();
+
+// require('./scripts/fix-library-paths.js');
+
+// loadCMakeVars(__dirname + '/../../../package/osx/build/CMakeCache.txt');
```

Comment:
Remove commented out experiments (can always look back in git history if needed again).